lib/utilities/locationConstraintCheck.js

```diff
@@ -0,0 +1,35 @@
+import { errors } from 'arsenal';
+
+import config from '../Config';
+import { BackendInfo } from '../api/apiUtils/object/BackendInfo';
+import constants from '../../constants';
+
+export default function locationConstraintCheck(request, metaHeaders,
```

Comment:
this function should not take a callback.  It is only doing synchronous actions.  I would make it a synchronous function that returns either backendInfo or an arsenal error.  If it returns an error, then from the api you call your cb with that error.